Update README.md (#2581)
idno/known: README.md

## Installation 

### Shared Host on  Known sites

> If you want to install on your own web space, we recommend [Reclaim Hosting](https://portal.reclaimhosting.com/aff.php?aff=013), which includes one-click Known installation.The one click installs an outdated version of 0.9.2 and then requires updating following the intructions below.  

Reviewer: ```suggestion
If you want to install on your own web space, we recommend [Reclaim Hosting](https://portal.reclaimhosting.com/aff.php?aff=013), which includes one-click Known installation. Currently this installs version 0.9.2, which then requires updating using the instructions below.  
```

Reviewer 2: Interestingly, GitHub gives me the button to merge these changes, but gives me a permission error when actually attempting to do so

Reviewer: I get the same. It's because we don't own @jgmac1106's branch. He needs to merge them!

Reviewer 2: Bit daft to show the button if you don't have access to perform the action. Basic UX
